iron-selectable.html

```diff
@@ -245,6 +245,12 @@
       if (this.selectedAttribute) {
         this.toggleAttribute(this.selectedAttribute, isSelected, item);
       }
+
+      if (isSelected) {
+        item.setAttribute('aria-selected', 'true');
+      } else {
+        item.removeAttribute('aria-selected');
+      }
```

Comment:
Should a menu/tab set `aria-selected` themselves (or `IronMenuBehavior`)? For example, `iron-pages` uses selectable and `aria-selected` doesn't make sense there.